interpreter/interpreter.go

```diff
@@ -213,22 +217,19 @@ func (i *exprInterpretable) resolveUnknown(unknown types.Unknown,
 		}
 	}
 	if !validIdent {
-		return false
+		return types.NewErr("Invalid identifier encountered.")
 	}
 	pkg := i.interpreter.packager
 	tp := i.interpreter.typeProvider
 	for _, id := range pkg.ResolveCandidateNames(identifier) {
 		if object, found := currActivation.ResolveName(id); found {
```

Comment:
Shadows previous "object" - is there a useful adjective to add?

Reply:
When used in the if block like this, the `object` is only visible within the `if`.